MediaAdminBundle/Controller/Api/MediaController.php

```diff
@@ -120,27 +121,28 @@ public function uploadAction($folderId, Request $request)
         $uploadedFile = $request->files->get('file');
         $saveMediaManager = $this->get('open_orchestra_media_admin.manager.save_media');
 
-        if ($uploadedFile && $filename = $saveMediaManager->getFilenameFromChunks($uploadedFile)) {
+        if ($uploadedFile && $uploadedFile = $saveMediaManager->getFilenameFromChunks($uploadedFile)) {
```

Comment:
@alavieille : you need to change the method name if it's now returning an UplodadFile instead of string. Something like getFileFromChunks ?